.github/workflows/enable-auto-merge.yml

```
@@ -0,0 +1,24 @@
+permissions: {}
+
+jobs:
+  enable-auto-merge:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'dependencies')
+    permissions:
+      pull-requests: write
+    env:
+      PR_NUMBER: ${{ github.event.pull_request.number }}
+    steps:
+      if: contains(github.event.pull_request.labels.*.name, 'patch') || contains(github.event.pull_request.labels.*.name, 'minor')
```

actionlint (1.7.4)

"steps" section must be sequence node but got mapping node with "!!map" tag

(syntax-check)